### PR TITLE
user API updated

### PR DESCRIPTION
**Pull request by fstodulski**

### CODE REVIEW

File: src/components/UserSettingsModal/UserSettingsModal.tsx [hunk 1]
```diff
@@ -23,23 +31,23 @@ export class UserSettingsModal extends React.Component<
...
+      >
+        <div
+          className="user-settings-modal__background"
+          onClick={this.props.onClick}
+        />
         <div className="user-settings-modal__box">
           <figure className="user-settings-modal__avatar">
             <img
-              src="https://pbs.twimg.com/profile_images/1013798240683266048/zRim1x6M_400x400.jpg"
+              src={host + this.props.stores.assets.avatar.filePath}
```
Comment (stanimirtt) [inline src/components/UserSettingsModal/UserSettingsModal.tsx:50]: Add helper for get the url.

File: src/components/UserSettingsModal/UserSettingsModal.scss [hunk 2]
```diff
@@ -10,9 +10,35 @@
...
+  &--is-open {
+    opacity: 1;
+    pointer-events: auto;
+
+    .user-settings-modal {
+      &__box {
+        transform: translateY(0%);
+      }
+    }
+
```
Comment (stanimirtt) [inline src/components/UserSettingsModal/UserSettingsModal.scss]: Remove new line.,

**Review by stanimirtt — APPROVED** [review]

File: src/components/UserSettingsModal/UserSettingsModal.tsx (hunk 1 above)
Comment (fstodulski) [reply src/components/UserSettingsModal/UserSettingsModal.tsx:50]: It's for now. I will return from backend hostName with path as one;

File: src/services/user.ts [hunk 3]
```diff
@@ -1,6 +1,20 @@
+import { IUser } from './../interfaces/user';
```
Comment (suezzo) [inline src/services/user.ts]: `import { IUser } from './../interfaces` is possible

File: src/components/Common/Inputs/DefaultInput/DefaultInput.tsx [hunk 4]
```diff
@@ -3,10 +3,13 @@ import './DefaultInput.scss';
 import { IDefaultInput } from '../../../../interfaces/DefaultInput';
 import { validateInput, IRegExpTestResult } from '../helpers/validation';
 import { encrypt } from '../../../../helpers/Encrypting';
-import { Fetch } from '../../../../helpers/fetch';
-import { env } from '../../../../env';
+import { inject, observer } from 'mobx-react';
+import { IStores, IUser } from '../../../../interfaces';
+import { UserDetailsService } from '../../../../services/user';
```
Comment (suezzo) [inline src/components/Common/Inputs/DefaultInput/DefaultInput.tsx]: `IDefaultInput` should be imported at the 7th line.

**Review by suezzo — APPROVED** [review]

**Pull request merged**